### ISSUE TITLE
Delegate community management

### ISSUE THREAD

**Issue by Changaco** [liberapay/salon#29]
Running our [Diaspora*](https://framasphere.org/u/liberapay), [Twitter](https://twitter.com/liberapaye), and [Facebook](https://www.facebook.com/Liberapay-472128142945598/) accounts is taking a lot of my time. It would be great if one or several people stepped up to help with this.

**Comment by Changaco** [comment]
It would also be great if someone livened up our subreddit: [/r/liberapay](https://www.reddit.com/r/liberapay/).

**Comment by Changaco** [comment]
Some reading for whomever ends up running our Twitter account: [How to grow your Twitter audience, in 3 easy steps](http://www.monstaftp.com/blog/how-to-grow-your-twitter-audience-in-3-easy-steps).

**Comment by Changaco** [comment]
Found something that might be useful to improve our Facebook page: https://nonprofits.fb.com/

**Comment by MartinDelille** [comment]
I could help for the facebook page which is the social network I'm most connected with.

**Comment by Changaco** [comment]
Thanks! I've added you as an editor, can you confirm that you now have access?

**Comment by Changaco** [comment]
We could use more followers on Twitter, our tweets don't even appear [in search](https://twitter.com/search?f=tweets&vertical=default&q=liberapay%20OR%20liberapaye&src=typd) until I raise their importance by retweeting them from my personal account.

**Comment by MartinDelille** [comment]
Yes I received the invitation, I'm now admin, I invited part of my friend so now we almost triple the page like number :-)

**Comment by KindlyFire** [comment]
I'm willing to do some community management (Twitter and Reddit) if it's still needed :)

Usernames: KindlyFire on both

**Comment by Changaco** [comment]
Yes it is, especially on Reddit.

The current thread for news we want to relay is #34, and older news can be found on our Diaspora* account if you want to make a catch-up post on Reddit.

I haven't posted any guidelines before, here are the ones that come to mind:

- we don't follow accounts of individuals, only projects/organizations
- we generally don't interact with controversial users (https://github.com/liberapay/liberapay.org/issues/11) and we definitely don't advertise them
- when in doubt, ask me first, here or in IRC

You don't need any permissions to get started on Reddit, but I'll add you as a moderator anyway. For Twitter I'll look into giving you access via TweetDeck.

**Comment by Changaco** [comment]
![spectacle h15196](https://cloud.githubusercontent.com/assets/1581590/15246303/06471720-190c-11e6-9fda-a04ea6e446f1.png)

---

![spectacle m15196](https://cloud.githubusercontent.com/assets/1581590/15246274/d20a7344-190b-11e6-8d5c-7d22dcd24b4a.png)

**Comment by MartinDelille** [comment]
I haven't been active yet on facebook but looking at the last post, I remove the like from Liberapay and add mine. It'll be nice that people from the team like/share post from social media to increase the range of the publications!

**Comment by jorgesumle** [comment]
> Running our Diaspora*, Twitter, and Facebook accounts is taking a lot of my time. It would be great if one or several people stepped up to help with this.

Do you still need help with Diaspora*. I could help with that.

**Comment by Changaco** [comment]
I still need help with everything. Diaspora* doesn't have multi-user accounts so we'll need to set up a password sharing mechanism.

**Comment by jorgesumle** [comment]
![bildschirmfoto vom 2017-04-27 01-19-53](https://cloud.githubusercontent.com/assets/14212780/25461336/b7247e40-2ae8-11e7-94fa-c585757af1db.png)
> It would also be useful to say publicly who currently manages the Liberapay Diaspora* account, such as fellows at FSFE do.

I thought it was possible. Anyway, if there is still necessary a password sharing mechanism, I'd recommend to sending me the email with the password encrypted. It would also be useful to say publicly who currently manages the Liberapay Diaspora* account, such as guys at FSFE do, for transparency.

**Comment by Changaco** [comment]
We now have a tool for password sharing (#130) and @jorgesumle has been given access to our Diaspora* account through that.

**Comment by trebmuh** [comment]
I'm happy to help with mastodon and diaspora if needed. Just let me know if so.

**Comment by Changaco** [comment]
@trebmuh I need an email address to invite you.

**Comment by trebmuh** [comment]
(done on IRC)

**Comment by mattbk** [comment]
I can help on Twitter if still needed.

**Comment by MartinDelille** [comment]
I created a telegram channel: https://t.me/liberapay

A PR is on its way: https://github.com/liberapay/liberapay.com/pull/1218

**Comment by MartinDelille** [comment]
I'll be happy to get back admin of the facebook page also!

**Comment by kindlyfire** [comment]
@MartinDelille could you add me as admin to the telegram channel, so that I can post too ? I joined it yesterday.

**Comment by MartinDelille** [comment]
@KindlyFire I sent you a PM on telegram just to check that is was you.

**Comment by MartinDelille** [comment]
I propose also a telegram group so user can reach the team: https://t.me/joinchat/HZKyvg8F5-ncY5c1ukzDbA

We currently use the same logo, but it would be cool to have two different logo to differenciate them (a simple color shift would do the job).
